### DISCUSSION TITLE
Option for Sound to use System Default

### DISCUSSION THREAD

**Discussion by Crazycolbster** [OpenRCT2/OpenRCT2#19639] [Feature requests]
For context, I was working on a plugin at the library, using Linux Mint 21.1. Despite enabling my Bluetooth headphones on the OS level, OpenRCT2 stuck to its default of using my speakers when it started. The sounds of that chain lift playing out my speaker was a little embarrassing at the library where we're supposed to be quiet. Could we have an option to have the game sound play out of whatever the default speaker the computer is using? 

Don't get me wrong, by the way, I super appreciate being able to have the option to select which device OpenRCT2 plays its sound out of.

**Comment by kash0lt** [comment]
Seems a reasonable request. I had the same experience, though not in a library setting, but still, the sound effects did not follow the chosen system default.
